src/repositories/projectRepository.ts

```diff
@@ -28,21 +28,9 @@ export const projectsWithoutUpdateAfterTimeFrame = async (date: Date) => {
       'project.projectVerificationForm',
       'projectVerificationForm',
     )
-    .innerJoinAndMapOne(
-      'project.projectUpdate',
-      ProjectUpdate,
-      'projectUpdate',
-      `project.id = projectUpdate.projectId AND projectUpdate.id = (
-        SELECT project_update.id
-        FROM project_update
-        WHERE project_update."projectId" = project.id
-        ORDER BY project_update.id DESC
-        LIMIT 1
-      )`,
-    )
     .where('project.isImported = false')
     .andWhere('project.verified = true')
-    .andWhere('projectUpdate.createdAt < :badgeRevokingDate', {
+    .andWhere('project.updatedAt < :badgeRevokingDate', {
       badgeRevokingDate: date,
```

Comment:
Simplified the query as any update on the project should relieve them from being revoked (description, title)

And also if they add any projectUpdate entity o modify the current one. It would automatically also update the updatedAt field on the project. So I think using updatedAt is good in this case.